src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -57,6 +57,55 @@
  * <pre>
  * &lt;module name=&quot;InnerAssignment"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass {
+ *
+ *   void foo() {
+ *     int a, b;
+ *     a = b = 5; // violation; assignment to each variable should be in a separate statement
```

Comment:
`violation;` should be comma instead of semicolon

Reply:
Sure. Done